x-pack/plugins/lens/public/xy_visualization/visualization.tsx

```diff
@@ -166,9 +170,55 @@ export const getXyVisualization = ({
       return { groups: [] };
     }
 
+    const layerContainsSplits = Boolean(layer.splitAccessor);
+
     const datasource = frame.datasourceLayers[layer.layerId];
 
-    const sortedAccessors = getSortedAccessors(datasource, layer);
+    const sortedAccessors: string[] = getSortedAccessors(datasource, layer);
```

Comment:
Could we extract the logic of assigning `mappedAccessors` to function for readability?